x-pack/plugins/monitoring/public/lib/setup_mode.tsx

```diff
@@ -212,3 +209,12 @@ export const isInSetupMode = () => {
   const globalState = $injector.get('globalState');
   return globalState.inSetupMode;
 };
+
+export const isSetupModeFeatureEnabled = (feature: SetupModeFeature) => {
```

Comment:
nit: Maybe we can also add a check here something like:
```javascript
if (!setupModeState.enabled) {
  return false;
}
```
So we don't always have to check for `setupMode && setupMode.enabled` before checking the feature

Reply:
I like this idea! It will make those checks a little better, I'll clean that up